src/components/common/AccountNote.js

```diff
@@ -20,17 +21,19 @@ const Container = styled.div`
 const AccountNote = () => (
 
     <Container>
-        <p><Translate id='createAccount.note.one'/></p>
+        <p><Translate id='createAccount.note.canContain'/></p>
         <ul>
-            <li><Translate id='createAccount.note.two'/></li>
-            <li><Translate id='createAccount.note.three'/></li>
-            <li><Translate id='createAccount.note.four'/></li>
+            <li><Translate id='createAccount.note.lowercase'/></li>
+            <li><Translate id='createAccount.note.digits'/></li>
+            <li><Translate id='createAccount.note.separators'/></li>
         </ul>
-        <p><Translate id='createAccount.note.five'/></p>
+        <p><Translate id='createAccount.note.cannotContain'/></p>
         <ul>
-            <li><Translate id='createAccount.note.six'/></li>
+            <li><Translate id='createAccount.note.characters'/></li>
+            <li><Translate id='createAccount.note.minCharacters'/></li>
+            <li><Translate id='createAccount.note.maxCharacters' data={{ accountSuffix: ACCOUNT_ID_SUFFIX }}/></li>
```

Comment:
New keys! 🔑 Needs to be translated to other languages.

Reply:
@picturepan2 when you have a chance can you please add translation for min and max chars?